app/main.go

```diff
@@ -1,25 +1,45 @@
 package main
 
 import (
-	"encoding/json"
+	"html/template"
 	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 type response struct {
 	count int
 }
 
+var (
+	dummyCounter = prometheus.NewCounter(
```

Comment:
we probably don't want this going into the pre-workshop code, right?

Reply:
yep! your right